[2025-05-28T09:39:47.625+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: register_best_model_from_mlflow.select_best_model_from_mlflow manual__2025-05-28T09:39:44.766328+00:00 [queued]>
[2025-05-28T09:39:47.642+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: register_best_model_from_mlflow.select_best_model_from_mlflow manual__2025-05-28T09:39:44.766328+00:00 [queued]>
[2025-05-28T09:39:47.643+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-05-28T09:39:47.660+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): select_best_model_from_mlflow> on 2025-05-28 09:39:44.766328+00:00
[2025-05-28T09:39:47.677+0000] {standard_task_runner.py:60} INFO - Started process 17206 to run task
[2025-05-28T09:39:47.683+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'register_best_model_from_mlflow', 'select_best_model_from_mlflow', 'manual__2025-05-28T09:39:44.766328+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/register_best_model_from_mlflow.py', '--cfg-path', '/tmp/tmpk1iq1ath']
[2025-05-28T09:39:47.687+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask select_best_model_from_mlflow
[2025-05-28T09:39:47.776+0000] {task_command.py:423} INFO - Running <TaskInstance: register_best_model_from_mlflow.select_best_model_from_mlflow manual__2025-05-28T09:39:44.766328+00:00 [running]> on host bbc28cef36b5
[2025-05-28T09:39:47.852+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='register_best_model_from_mlflow' AIRFLOW_CTX_TASK_ID='select_best_model_from_mlflow' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T09:39:44.766328+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-28T09:39:44.766328+00:00'
[2025-05-28T09:39:47.855+0000] {logging_mixin.py:188} INFO - üîÑ Using updated DAG version ‚Äî logging active.
[2025-05-28T09:39:47.860+0000] {file_store.py:331} WARNING - Malformed experiment '1'. Detailed error Yaml file '/app/mlruns/1/meta.yaml' does not exist.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/mlflow/store/tracking/file_store.py", line 327, in search_experiments
    exp = self._get_experiment(exp_id, view_type)
  File "/home/airflow/.local/lib/python3.8/site-packages/mlflow/store/tracking/file_store.py", line 421, in _get_experiment
    meta = FileStore._read_yaml(experiment_dir, FileStore.META_DATA_FILE_NAME)
  File "/home/airflow/.local/lib/python3.8/site-packages/mlflow/store/tracking/file_store.py", line 1367, in _read_yaml
    return _read_helper(root, file_name, attempts_remaining=retries)
  File "/home/airflow/.local/lib/python3.8/site-packages/mlflow/store/tracking/file_store.py", line 1360, in _read_helper
    result = read_yaml(root, file_name)
  File "/home/airflow/.local/lib/python3.8/site-packages/mlflow/utils/file_utils.py", line 309, in read_yaml
    raise MissingConfigException(f"Yaml file '{file_path}' does not exist.")
mlflow.exceptions.MissingConfigException: Yaml file '/app/mlruns/1/meta.yaml' does not exist.
[2025-05-28T09:39:48.283+0000] {logging_mixin.py:188} INFO - üèÜ –õ—É—á—à–∞—è –º–æ–¥–µ–ª—å: run_id=7957650d20b148569979546d20ed473f, f1_score=0.5857320280537698
[2025-05-28T09:39:48.300+0000] {logging_mixin.py:188} INFO - ‚úÖ –ú–æ–¥–µ–ª—å —Å–∫–æ–ø–∏—Ä–æ–≤–∞–Ω–∞: /app/artifacts/best_model.pkl
[2025-05-28T09:39:49.915+0000] {logging_mixin.py:188} INFO - üìå –í—ã–ø–æ–ª–Ω—è–µ–º git commit...
[2025-05-28T09:39:50.039+0000] {logging_mixin.py:188} INFO - üîß Git commit stdout: [develop a9e2642] Register best model from MLflow
 1 file changed, 2 insertions(+), 2 deletions(-)
[2025-05-28T09:39:50.041+0000] {logging_mixin.py:188} INFO - üîß Git commit stderr: 
[2025-05-28T09:39:57.887+0000] {logging_mixin.py:188} INFO - üöÄ –ú–æ–¥–µ–ª—å –¥–æ–±–∞–≤–ª–µ–Ω–∞ –≤ DVC –∏ –∑–∞–ø—É—à–µ–Ω–∞
[2025-05-28T09:39:57.891+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-05-28T09:39:57.931+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=register_best_model_from_mlflow, task_id=select_best_model_from_mlflow, execution_date=20250528T093944, start_date=20250528T093947, end_date=20250528T093957
[2025-05-28T09:39:58.041+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T09:39:58.129+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
